protofire/omen-exchange
# Proposal Details

closes #2190 
How to test:
Because data is not connected we can only test styles so we need to make sure that designs are same as on figma:
- To access page go to http://localhost:3000/#/proposal-details-page
- Liquidity rewards section(mobile and desktop) https://www.figma.com/file/zVniQrYgkj8AfQs5zNeBrS/Components-FINAL?node-id=300%3A1290
- Whole page(mobile and desktop https://www.figma.com/file/PhYs1uw1LgqpvriqHDPl7k/Playground-FINAL?node-id=402%3A12460
